pkg/options/describe_test.go

```diff
@@ -0,0 +1,433 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
I am reaching out to other devs on `tkn` who are more familiar with `interactive` testing approaches to see if there would be better approaches than this for testing this feature. I am not fully convinced these are really covering this feature. 

I realize this approach is used with other features, but we might want to rethink it as a viable approach for these kinds of tests.

Reply:
`/pkg/options/describe_test.go` is based on `/pkg/options/logs_test.go` since Describe Options and Logs Options are similer, I kept the tests for describe.go similer to test present for logs.go go in `options` package.

Reply:
Yes, it is my opinion that those should have never been accepted as tests and am not sure why they were. Basically, you can see the code coverage [here](https://github.com/tektoncd/cli/pull/890#issuecomment-618881461) and click on the link for a further break down. The tests aren't covering anything but rather trying to mock the behavior, which isn't really helpful when someone changes the code later on as there will be no precedent for what should still work after a change. So basically we need a testing pattern for interactive testing. I would suggest moving this tests into the unit tests for subcommands (i.e. `/pkg/cmd/<resoure>/describe_test.go` so the logic being added is also covered in describe command code as well. I'll also try and think of a testing approach here.

Reply:
I think the `/pkg/option/describe.go` is just for asking which resource to describe, so we should only test if the correct resources are been given in options and choosing one is setting the correct resource name in `options.DescribeOptions` . I think the tests should be here only in /pkg/options/ because the test for if a resource is being described properly or the description based tests are in the unit tests of subcommands. Here we are choosing a resource so the tests for the 'choosing' part should be here.
Yes, I agree with the coverage part as tests for FuzzyAsk are not there. We would need to include them.

Reply:
Ah, my apologies for the confusion in all this. I see what you are saying about testing `Ask` and think these are valid.

Reply:
> Yes, I agree with the coverage part as tests for FuzzyAsk are not there. We would need to include them.

If you do have thoughts around testing `FuzzyAsk`, I think that would be great though. That would be my only other ask for this pr.

Reply:
Something to potentially consider is how some of the funcs from `fuzzyfinder` are tested:

https://github.com/ktr0731/go-fuzzyfinder/blob/d856dcab9233fa197ce52a4daa33c4b6dca85248/fuzzyfinder_test.go#L258

Reply:
`FuzzyAsk` tests are not implemented for `logs` also :P , I don't know why I think it would be challenging for me but I will give it a try.